floyd/cli/utils.py

```diff
@@ -116,10 +116,9 @@ def normalize_job_name(raw_job_name, default_username='', default_project_name='
 
     if use_config:
         default_project_name = default_project_name or current_project_name()
+        namespace = default_username or current_project_namespace()
```

Comment:
current_project_namespace depends on config being available. you might want to initialize namespace before `if use_config` branch, then call current_project_namespace inside the if branch

Reply:
I'm not sure if my last commit is fixing this in the right way, because I had also to fix the test.